admin/console/other-useful-commands.rst

```diff
@@ -13,29 +13,47 @@ This will also show errors that might appear within that process.
 
    >> Channel.fetch
 
-Reprocess unprocessable mails
------------------------------
+Reprocess Unprocessable Emails
```

Comment:
We should change the naming. Unprocessable mails is not used any more, we now talk about "failed emails" instead.

Reply:
I replaced it in the text as well but kept one "unprocessable" (in brackets) so that it can still be found by searching "unprocessable emails"